### PR TITLE
Exercises

### PR DESCRIPTION
**Pull request by shaypheraddison**

### CODE REVIEW

File: exercises/exercise1.js [hunk 1]
```diff
@@ -1 +1,18 @@
...
+    appleJuice: 3.99,
+    cups: 12.50,
+    dogFood: 24.99
+};
+
+for (item in receipt) {
+    console.log(item);
+    console.log(receipt[item]);
+}
+const totalSum = receipt.apples + receipt.milk + receipt.appleJuice + receipt.cups + receipt.dogFood;
```
Comment (resoluteCoder) [inline exercises/exercise1.js]: Can you find a better way to add the sum?

File: exercises/exercise1.js [hunk 2]
```diff
@@ -1 +1,18 @@
...
+const receipt = {
+    apples: 2.45,
+    milk: 4.25,
+    appleJuice: 3.99,
+    cups: 12.50,
+    dogFood: 24.99
+};
+
+for (item in receipt) {
+    console.log(item);
```
Comment (resoluteCoder) [inline exercises/exercise1.js:15]: Try string interpolation

File: exercises/exercise2.js [hunk 3]
```diff
@@ -9,3 +9,12 @@
 // # print(king_of_hearts)
 // # print(king_of_spades)
 
+const suits = ["Hearts", "Clubs", "Diamonds", "Spades"];
+const faces = ["Ace", "King", "Queen", "Jack", "10", "9", "8", "7", "6", "5", "4", "3", "2"];
+console.log(suits);
+console.log(faces);
+for (let suit of suits) {
+    for (let face of faces) {
+        console.log(face + " of " + suit);
```
Comment (resoluteCoder) [inline exercises/exercise2.js]: This prints them out not creating a deck of cards with string values

File: exercises/exercise3.js [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+    suits: ["Hearts", "Clubs", "Diamonds", "Spades"]
+};
+
+for (card in cards) {
+    for (let face of cards.suits) {
+        for (let suit of cards.faces) {
+            console.log(suit + " of " + face);
+        }
+    }
+}
```
Comment (resoluteCoder) [inline exercises/exercise3.js]: This needs to a deck of cards where each card is an object. You are printing them.

File: exercises/exercise4.js [hunk 5]
```diff
@@ -5,4 +5,8 @@
 // #    *****
 // #   *******
 
+const stars = ["    *", "   ***", "  *****", " *******"];
 
+for (let star of stars) {
+    console.log(star);
```
Comment (resoluteCoder) [inline exercises/exercise4.js]: Negative ghost rider 😄

**Review by resoluteCoder — CHANGES_REQUESTED** [review]

File: exercises/exercise2.js (hunk 3 above)
Comment (shaypheraddison) [reply exercises/exercise2.js]: completed this and have pushed the commit

File: exercises/exercise1.js (hunk 2 above)
Comment (shaypheraddison) [reply exercises/exercise1.js:15]: added string interpolation and pushed commit

File: exercises/exercise3.js (hunk 4 above)
Comment (shaypheraddison) [reply exercises/exercise3.js]: Corrected to have each card as an object. pushed commit as well

File: exercises/exercise1.js (hunk 1 above)
Comment (shaypheraddison) [reply exercises/exercise1.js]: Added a more efficient way to calculate sum in new commit.

File: exercises/exercise3.js [hunk 6]
```diff
@@ -1,2 +1,12 @@
 // # create a deck of cards where the cards are a dictionary
 
+
+const faces = ["Ace", "King", "Queen", "Jack", "10", "9", "8", "7", "6", "5", "4", "3", "2"];
+const suits = ["Hearts", "Clubs", "Diamonds", "Spades"];
+
+for (let face of faces) {
+    for (let suit of suits) {
+        newDeck = {Face: face, Suit: suit};
```
Comment (resoluteCoder) [inline exercises/exercise3.js]: Take a close look at this line. How many `newDeck` will be created.  Will `newDeck` have all the cards in it?

**Review by resoluteCoder — CHANGES_REQUESTED** [review]

File: exercises/exercise4.js (hunk 5 above)
Comment (shaypheraddison) [reply exercises/exercise4.js]: This has been properly completed. Forgot to respond to this.

File: exercises/exercise3.js [hunk 7]
```diff
@@ -4,9 +4,9 @@
 const faces = ["Ace", "King", "Queen", "Jack", "10", "9", "8", "7", "6", "5", "4", "3", "2"];
 const suits = ["Hearts", "Clubs", "Diamonds", "Spades"];
 
+var newDeck = [];
```
Comment (resoluteCoder) [inline exercises/exercise3.js]: Why var?

File: exercises/exercise3.js [hunk 8]
```diff
@@ -4,9 +4,9 @@
...
 for (let face of faces) {
     for (let suit of suits) {
-        newDeck = {Face: face, Suit: suit};
-        console.log(newDeck);
+        newDeck.push({Face: face, Suit: suit});
     }
-}
+} console.log(newDeck);
```
Comment (resoluteCoder) [inline exercises/exercise3.js]: log on new line? 😄

**Review by resoluteCoder — APPROVED** [review]

**Comment by resoluteCoder** [comment]
Looks good just left a couple of comments but it can be merged that way. Good job!

**Pull request merged**